make/photon/prepare/utils/jobservice.py

```diff
@@ -18,7 +18,8 @@ def prepare_job_service(config_dict):
 
     # Job log is stored in data dir
     job_log_dir = os.path.join('/data', "job_logs")
-    prepare_config_dir(job_log_dir)
+    file_path = prepare_config_dir(job_log_dir)
```

Comment:
this is trivial but why assign another var instead of reusing `job_log_dir`

Reply:
I think `prepare_config_dir` function will return a value which is the real path, although it is the same as the original one if only one arg passed.